src/test/java/uk/ac/cam/cl/dtg/isaac/api/AuthenticationFacadeIT.java

```diff
@@ -0,0 +1,42 @@
+package uk.ac.cam.cl.dtg.isaac.api;
+
+import jakarta.servlet.http.Cookie;
```

Comment:
Unused imports. You can configure your IDE to get rid of these automatically.

Reply:
I'm pretty sure I configured my IDE to do that last time this came up but it still seems a bit unreliable. I'll trigger the optimisation manually...